docs/home/howto/LOGGING.md

```diff
@@ -46,3 +49,28 @@ specification:
     repo: /var/lib/elasticsearch-snapshots
     logs: /var/log/elasticsearch
 ```
+
+### How to export Elasticsearch data to csv format
+Since v0.8 Epiphany provide posibility to export data from Elasticsearch to CSV using Logstash *(logstash-oss v7.8.1*) along with *logstash-input-elasticsearch (v4.6.2)* and *logstash-output-csv (v3.0.8)* plugin.
+
+To install Logstash in your cluster add **logstash** to feature mapping for *logging, opendistro_for_elasticsearch* or *elasticsearch* group.
+
+`Note: Logstash is disabled by default after installation.`
+
+Epiphany provide a basic configuration file `(logstash-export.conf.temp)` as template for your data export.
```

Comment:
Does it have to be `.temp`? Is this some default way of naming it? If not then can we change it somehow? I'm asking because from the first look it's like a temporary file and this will probably trigger more people. :)

Reply:
@sk4zuzu Documentation stands clearly that it's a template but I understand this can cause a misunderstanding. Any suggestions? (maybe .draft,.template)

Reply:
I'd go with `.template` - it's a little longer, but does not trigger certain people... 🤭